docker/compose.testing.yaml

```
@@ -40,11 +40,11 @@
     # https://docs.docker.com/compose/environment-variables/
     # https://docs.docker.com/reference/compose-file/services/#environment
     environment:
-      - MINIO_BROWSER=${MINIO_TEST_BROWSER:?error}
+      - MINIO_BROWSER=${MINIO_BROWSER:?error}
         # https://min.io/docs/minio/linux/reference/minio-server/settings/root-credentials.html#envvar.MINIO_ROOT_PASSWORD
```

yamllint (1.35.1)

[warning] comment not indented like content

(comments-indentation)